### PR TITLE
Update Chinese translation to 1.20.1 from 1.17.3 based on the existing translation

### PR DESCRIPTION
**Pull request by Genglxy**
update content:
Fixed the bug of full-width half-width colon, which would cause {:x} to be unrecognized by the program.
I merged my previous translations with the existing translation, partly based on them, and corrected a few incorrect expressions.
Fixed the problem of &.
Correct the layout, add spaces between Chinese and English, and add English contrast to the technical vocabulary with low frequency of use.
Add new translation content.
For Shader mul accuracy, since the exact translation of mul cannot be found, the original English text is used instead.
Added: Note the following explanation, Shader mul accuracy has been translated into 着色器乘法精度(shader multiplication accuracy).

Supplement: I just noticed that this translation is not based on #49. The previous 1.17.3 translation updated by the manager was created by another translator. Respect the achievements of the predecessors, in the following translations, I will refer to the translation of #49 and choose the best of the three.

### CODE REVIEW

File: resources/zh/cemu.po [hunk 1]
```diff
@@ -2021,88 +2594,281 @@ msgstr "DLC"
...
+
 #: wxGameList.cpp
 msgid "&Start"
-msgstr "启动"
+msgstr "&启动"
 
 #: wxGameList.cpp
 msgid "&Favorite"
-msgstr "收藏"
+msgstr "&收藏并置顶"
```
Comment (Fs00) [inline resources/zh/cemu.po:2613]: I don't think it's needed to explicitly say that the game will be put at the top of the list. Users will notice it by themselves.

File: resources/zh/cemu.po [hunk 2]
```diff
@@ -2021,88 +2594,281 @@ msgstr "DLC"
...
+msgid "&Launch title"
+msgstr "&载入档案"
+
+#: wxTitleManagerList.cpp
+msgid "&Open directory"
+msgstr "&打开目录"
+
+#: wxTitleManagerList.cpp
+msgid "Fix entry"
+msgstr "修复入口 | Fix entry"
```
Comment (Fs00) [inline resources/zh/cemu.po:2758]: If you prefer, you can translate it as "Fix issues" instead of "Fix entry" (it fits better imho). Then remove the English version.

File: resources/zh/cemu.po [hunk 3]
```diff
@@ -2021,88 +2594,281 @@ msgstr "DLC"
...
+msgid ""
+"Are you really sure that you want to delete the following folder:\n"
+"{}"
+msgstr "你真的确定要删除以下文件夹吗？"
+
+#: wxTitleManagerList.cpp
+msgid ""
+"Are you really sure that you want to delete the following file:\n"
+"{}"
+msgstr "你确定要删除以下文件吗？"
```
Comment (Fs00) [inline resources/zh/cemu.po:2774]: The {} are missing in these translations

File: resources/zh/cemu.po [hunk 4]
```diff
@@ -85,6 +57,122 @@ msgstr "输入内存地址"
...
+msgid "Select a file"
+msgstr "选择一个文件"
+
+#: CemuApp.cpp
+msgid "Select a directory"
+msgstr "选择一个目录"
+
+#: CemuApp.cpp
+msgid "base"
+msgstr "本体"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:74]: "本体" in Chinese is the abbreviation for "游戏本体", which means that the basic part of the game "excluding DLC and updates", which is the base game. So the change here is unnecessary.

File: resources/zh/cemu.po [hunk 5]
```diff
@@ -85,6 +57,122 @@ msgstr "输入内存地址"
...
+msgid "update"
+msgstr "更新档"
+
+#: CemuApp.cpp
+msgid "dlc"
+msgstr "DLC / 附加内容"
+
+#: CemuApp.cpp
+msgid "save"
+msgstr "存档"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:86]: 存档 has dual meanings of nouns and verbs in Chinese. By default, the Chinese context will treat it as a noun, which is Gamesave.

File: resources/zh/cemu.po [hunk 6]
```diff
@@ -85,6 +57,122 @@ msgstr "输入内存地址"
...
+msgid "Taiwan"
+msgstr "中国台湾"
+
+#: CemuApp.cpp
+msgid "Auto"
+msgstr "自动"
+
+#: CemuApp.cpp
+msgid "many"
+msgstr "many/多区域"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:122]: This is difficult to determine. "many" is sandwiched between regions and languages in the translation file. The former is the region in the game list, so it may be more appropriate to translate into multiple regions. In order to prevent translation errors, I attached English to ensure that translation problems will not occur. If the correct translation method is determined later, I will add it.

File: resources/zh/cemu.po [hunk 7]
```diff
@@ -1,68 +1,40 @@
...
 
-#: BreakpointWindow.cpp textureRelationWindow.cpp
+#: BreakpointWindow.cpp textureRelationWindow.cpp wxTitleManagerList.cpp
 msgid "Type"
 msgstr "类型"
 
 #: BreakpointWindow.cpp
 msgid "Comment"
-msgstr "注释"
+msgstr "注释 | Comment"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:33]: It's not really necessary. . . I added it earlier because I'm not sure whether the translation is accurate, I will delete it.

File: resources/zh/cemu.po [hunk 8]
```diff
@@ -137,20 +225,36 @@ msgid ""
...
+"Cemu 无法写入选择的 mlc 路径！\n"
+"你想尝试选择其他路径吗？"
+
+#: CemuApp.cpp DownloadGraphicPacksWindow.cpp DumpCtrl.cpp GameUpdateWindow.cpp
+#: GeneralSettings2.cpp gpu7ShaderCache.cpp helpers.cpp InputSettings.cpp
+#: MainWindow.cpp SaveImportWindow.cpp SaveTransfer.cpp TitleManager.cpp
+#: toolMemorySearcher.cpp VulkanCanvas.cpp wxCreateAccountDialog.cpp
+#: wxTitleManagerList.cpp
+msgid "Error"
+msgstr "错误 | Error"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:250]: It is indeed not particularly necessary here, I will delete it. . . Sorry for the troubles.

File: resources/zh/cemu.po [hunk 9]
```diff
@@ -212,59 +316,59 @@ msgstr "中止 (F5)"
 
 #: DebuggerWindow2.cpp
 msgid "Step Into (F11)"
-msgstr "逐步执行(Step Into) (F11)"
+msgstr "逐步执行 | Step Into (F11)"
 
 #: DebuggerWindow2.cpp
 msgid "Step Over (F10)"
-msgstr "逐程序執行(Step Over) (F10)"
+msgstr "逐程序执行 | Step Over (F10)"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:323]: of course! I haven't found a suitable translation method here, so I used English when I first translated. This is the work of the previous translator. I will replace it with a new translation, thank you very much!

File: resources/zh/cemu.po [hunk 10]
```diff
@@ -212,59 +316,59 @@ msgstr "中止 (F5)"
...
 
 #: DebuggerWindow2.cpp MainWindow.cpp
 msgid "&Options"
-msgstr "选项"
+msgstr "&选项"
 
 #: DebuggerWindow2.cpp
 msgid "&Registers"
-msgstr "寄存器"
+msgstr "&寄存器 | Registers"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:347]: OK

File: resources/zh/cemu.po [hunk 11]
```diff
@@ -212,59 +316,59 @@ msgstr "中止 (F5)"
...
 
 #: DebuggerWindow2.cpp
 msgid "&Memory Dump"
-msgstr "内存转储"
+msgstr "&内存转储"
 
 #: DebuggerWindow2.cpp
 msgid "&Breakpoints"
-msgstr "断点"
+msgstr "&断点 | Breakpoints"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:355]: of course.

File: resources/zh/cemu.po [hunk 12]
```diff
@@ -212,59 +316,59 @@ msgstr "中止 (F5)"
...
+msgstr "&窗口"
 
 #: debugPPCThreadsWindow.cpp
 msgid "PPC threads"
 msgstr "PPC线程"
 
 #: debugPPCThreadsWindow.cpp
 msgid "Entry"
-msgstr "入口"
+msgstr "入口 | Entry"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:371]: Ok. I didn't understand before, I will only keep English.

File: resources/zh/cemu.po [hunk 13]
```diff
@@ -304,9 +408,9 @@ msgstr "线程名"
 
 #: debugPPCThreadsWindow.cpp
 msgid "GPR"
-msgstr "GPR"
+msgstr "通用暂存器"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:411]: Here I refer to the translated version 2 years ago, in that version it was translated as such. Change subsequently.

File: resources/zh/cemu.po [hunk 14]
```diff
@@ -284,11 +388,11 @@ msgstr "状态"
 
 #: debugPPCThreadsWindow.cpp
 msgid "Affinity"
-msgstr "Affinity"
+msgstr "相似 | Affinity"
 
 #: debugPPCThreadsWindow.cpp
 msgid "Priority"
-msgstr "优先级"
+msgstr "优先级 | Priority"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:395]: Same as the previous one, these entries were translated in the translation file two years ago, so I used them earlier. Modified subsequently.

File: resources/zh/cemu.po [hunk 15]
```diff
@@ -452,76 +555,35 @@ msgstr "设置CPU模拟模式"
...
 
 #: GameProfileWindow.cpp
 msgid ""
 "EXPERT OPTION\n"
 "Set the maximum thread slice runtime (in virtual cycles)"
 msgstr ""
-"专家选项\n"
-"Set the maximum thread slice runtime (in virtual cycles)"
+"专业选项 ⚠\n"
+"设置最大线程量运行（以虚拟周期为单位）"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:566]: I really didn't know much before. 😂This really needs me to think about how to translate it correctly. . . I will check some more information (because the content you posted is understandable, but it is still machine translation), and I will submit it with other changes after consideration. Thank you very much.

File: resources/zh/cemu.po [hunk 16]
```diff
@@ -452,76 +555,35 @@ msgstr "设置CPU模拟模式"
...
-msgstr ""
-"预编译着色器可减少着色器加载画面所需时间\n"
-"自动：将为AMD与Intel GPU启用而对NVIDIA GPU禁用以解决驱动错误\n"
-"建议选项：自动"
+msgstr "周期 | cycles"
 
 #: GameProfileWindow.cpp
 msgid "Shader mul accuracy"
-msgstr "着色器多重精度"
+msgstr "着色器乘法精度 | Shader mul accuracy"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:574]: It seems like a good idea, but I have been worried that it is still not correct enough, so I kept it before.

File: resources/zh/cemu.po [hunk 17]
```diff
@@ -569,49 +610,42 @@ msgid "Forces a given controller profile"
...
-
-#: GameUpdateWindow.cpp
-msgid "Invalid \"meta.xml\" file."
-msgstr "无效的\"meta.xml\"文件."
-
-#: GameUpdateWindow.cpp
-msgid "Can't find the \"title_id\" in the given \"meta.xml\" file."
-msgstr "无法在选择的\"meta.xml\"文件中找到“title_id”."
+msgid ""
+"It seems that there's a wrong title installed at the target location.\n"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:614]: Under normal circumstances, the similar function in the Chinese context is called the game manager, although it also manages dlc and updates. In view of the fact that "title" has appeared many times in this software, and it is different from "game" when used, it cannot be translated into a game manager here.
I realized this, which is why I translated it into 档案. File means a single file in Chinese, and 档案 usually means a set of archived and valuable files. Translation into 档案 is the result of my careful consideration. This is a more suitable translation that I can currently consider.

File: resources/zh/cemu.po [hunk 18]
```diff
@@ -1493,337 +1849,321 @@ msgstr "更新提醒"
...
 
 #: MainWindow.cpp
 msgid "&Vulkan validation layer (slow)"
-msgstr "Vulkan 验证层 (慢速)"
+msgstr "&Vulkan 验证层 (慢速)"
 
 #: MainWindow.cpp
 msgid "&Log PPC context for API"
-msgstr "为API记录PPC上下文"
+msgstr "&Log PPC context for API"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:2060]: Because I am not sure whether this translation is correct. Context in Chinese is usually only used in the article, which means the paragraph before and after, if the translation is correct, of course you can add it.
I will add it, and it won't be too late to change it back if someone gives feedback.

File: resources/zh/cemu.po [hunk 19]
```diff
@@ -1493,337 +1849,321 @@ msgstr "更新提醒"
...
 
 #: MainWindow.cpp
 msgid "&nlibcurl HTTP/HTTPS requests"
-msgstr "nlibcurl HTTP/HTTPS 请求"
+msgstr "&nlibcurl HTTP/HTTPS 请求"
 
 #: MainWindow.cpp
 msgid "&Logging"
-msgstr "记录"
+msgstr "&记录 Log"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:2076]: The following is not an English translation.
The meaning of "记录" and "log" is different in Chinese. "记录" means record, and here obviously means to record a log, so a more appropriate translation than "记录" is "记录日志".
In the Chinese context, "log" is used as a foreign noun and is often used instead of "日志", and "log" is used more frequently than "日志". So log is used here before, I will use "记录日志" to replace it later.

File: resources/zh/cemu.po [hunk 20]
```diff
@@ -1403,81 +1711,129 @@ msgstr "未输入设定档名称！"
...
+msgstr "搜寻 Wii 手柄 ..."
+
+#: InputSettings.cpp
+msgid "Searching for controllers..."
+msgstr "搜寻控制器 ..."
 
 #: LoggingWindow.cpp
 msgid "Logging window"
-msgstr "记录窗口"
+msgstr "Log 记录窗口"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:1740]: Log is used as a noun in my translation, I will change it to "日志记录窗口".

File: resources/zh/cemu.po (hunk 6 above)
[the remark this answers is not in the document]
Comment (Fs00) [reply resources/zh/cemu.po:122]: I will ask to the Cemu devs.

File: resources/zh/cemu.po [hunk 21]
```diff
@@ -1833,27 +2173,121 @@ msgstr "大小"
...
 
 #: RegisterWindow.cpp
 msgid "&Decrement"
-msgstr "减量"
+msgstr "&减少"
 
 #: RegisterWindow.cpp
 msgid "&Copy"
-msgstr "复制"
+msgstr "&复制 | Copy"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:2188]: Sorry😅

File: resources/zh/cemu.po [hunk 22]
```diff
@@ -1907,9 +2341,125 @@ msgstr "仅显示活动"
...
+msgid "Transfer"
+msgstr "转移"
+
+#: TitleManager.cpp
+msgid "Transfers the save entry to another persistent account id"
+msgstr "将存档条目转移到另一个 Persistent 帐户 ID 中"
+
+#: TitleManager.cpp
+msgid "Irrevocable delete the save entry "
+msgstr "删除存档条目（无法撤销） "
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:2386]: All right!

File: resources/zh/cemu.po (hunk 17 above)
[the remark this answers is not in the document]
Comment (Fs00) [reply resources/zh/cemu.po:614]: Understood. But considering a "title" as a "a set of archived and valuable files" is a bit too generic for this scenario.

File: resources/zh/cemu.po [hunk 23]
```diff
@@ -1907,9 +2341,125 @@ msgstr "仅显示活动"
...
+msgid "Export"
+msgstr "导出"
+
+#: TitleManager.cpp
+msgid "Exports the selected save entry as zip file"
+msgstr "将选定的存档条目导出为 zip 文件"
+
+#: TitleManager.cpp
+msgid "Found {} titles, {} updates, {} DLCs and {} save entries"
+msgstr "找到 {} 个游戏，{} 个更新档， {} 个 DLC 和 {} 个存档条目"
```
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:2406]: of course. I paid special attention to it when translating before.

File: resources/zh/cemu.po (hunk 19 above)
[the remark this answers is not in the document]
Comment (Fs00) [reply resources/zh/cemu.po:2076]: Ok. The original string seemed ok to me because [Microsoft Terminology Database](https://www.microsoft.com/en-us/language/Search?&searchTerm=log&langID=124&Source=true&productid=0) translates log into 记录 (or 日志).
"Record" in English is a synonym for "log" so I find it acceptable. But feel free to use "记录日志" if it sounds more appropriate.

File: resources/zh/cemu.po (hunk 1 above)
Comment (Genglxy) [reply resources/zh/cemu.po:2613]: In Chinese usage habits, the function "Favorite" usually puts the collected things in a separate collection folder, and the Favorite function like cemu is generally called topping in Chinese software —— very straightforward. So here I  translated it as "Favorite and put on top". If you still think it is redundant, I can delete it.

File: resources/zh/cemu.po (hunk 2 above)
Comment (Genglxy) [reply resources/zh/cemu.po:2758]: Of course, I couldn't find a more suitable translation before, and I will change it in a later submission.

File: resources/zh/cemu.po (hunk 3 above)
Comment (Genglxy) [reply resources/zh/cemu.po:2774]: I'm very sorry,  I was negligent. Will be modified in subsequent submissions.

Comment (Genglxy) [reply resources/zh/cemu.po:2774]: Thank you very much for your comments and will submit it later.

File: resources/zh/cemu.po (hunk 6 above)
[the remark this answers is not in the document]
Comment (Fs00) [reply resources/zh/cemu.po:122]: A Cemu dev told me it refers to multi-region, so your guess is correct. Remove the original "many" the next time you edit the file.
Thanks for having made me aware of that 😀

File: resources/zh/cemu.po (hunk 15 above)
[the remark this answers is not in the document]
Comment (Fs00) [reply resources/zh/cemu.po:566]: It might come in handy the original English text that explains thread quantum:
> The QuantumLength property defines the number of clock ticks per quantum. A quantum is a unit of execution time that the scheduler is allowed to give to an application before switching to other applications. When a thread runs one quantum, the kernel preempts it and moves it to the end of a queue for applications with equal priorities. The actual length of a thread's quantum varies across different Windows platforms.

File: resources/zh/cemu.po (hunk 17 above)
[the remark this answers is not in the document]
Comment (Genglxy) [reply resources/zh/cemu.po:615]: In a sense, "档案" is indeed broader. It can mean "publication", but of course it is not accurate enough. The "档案" in the vocabulary that can be found at present should be a relatively localized translation, and it can also allow users to understand the function of this title manager. If there is a better translation later, it will definitely be changed.

**Comment by Genglxy** [comment]
> Thanks! I've pointed out some errors (and potential ones) in your work.
> Be aware that most of my comments are based on the results of Google Translate/DeepL, so there can be some interpretation mistakes in the review.
> In addition to what I pointed out, I would ask you to keep the casing of "MLC" consistent (either all uppercase or all lowercase). Keep "mlc01" as it is.

I have revised the suggestions you made (except for Favorite, I think "collect and put on top" is a more localized translation) and submitted a new file! If there is anything I can do or need me to do, please feel free to point it out. Thank you very much for your efforts.

**Comment by Genglxy** [comment]
> Thanks! I've pointed out some errors (and potential ones) in your work.
> Be aware that most of my comments are based on the results of Google Translate/DeepL, so there can be some interpretation mistakes in the review.
> In addition to what I pointed out, I would ask you to keep the casing of "MLC" consistent (either all uppercase or all lowercase). Keep "mlc01" as it is.
Is there any problem with my operation? I noticed that the "Changes requested: 1 review requesting changes" at the bottom of the page did not disappear. Will this affect the merger? Is there anything else I need to do? I don't know GitHub, and I haven't found any related solutions on the Internet.

**Comment by Fs00** [comment]
> Is there any problem with my operation? I noticed that the "Changes requested: 1 review requesting changes" at the bottom of the page did not disappear. Will this affect the merger? Is there anything else I need to do? I don't know GitHub, and I haven't found any related solutions on the Internet.

Don't worry! That's because I haven't reviewed your latest changes yet (and therefore GitHub continues to show the result of my first review).
I'll review your fixes in the next few days (next weekend at worst).

**Comment by Genglxy** [comment]
> > Is there any problem with my operation? I noticed that the "Changes requested: 1 review requesting changes" at the bottom of the page did not disappear. Will this affect the merger? Is there anything else I need to do? I don't know GitHub, and I haven't found any related solutions on the Internet.
> 
> Don't worry! That's because I haven't reviewed your latest changes yet (and therefore GitHub continues to show the result of my first review).
> I'll review your fixes in the next few days (next weekend at worst).

No problem, take your time.

File: resources/zh/cemu.po [hunk 24]
```diff
@@ -452,76 +555,36 @@ msgstr "设置CPU模拟模式"
 
 #: GameProfileWindow.cpp
 msgid "Thread quantum"
-msgstr "Thread quantum"
+msgstr "线程量子长度为"
```
Comment (Fs00) [inline resources/zh/cemu.po:558]: This gets translated as "The thread quantum length is" by GTranslate/DeepL, is that true?
If it is, I think it's better to keep the style of the original string (so something like "thread quantum length")

File: resources/zh/cemu.po [hunk 25]
```diff
@@ -452,76 +555,36 @@ msgstr "设置CPU模拟模式"
...
 #: GameProfileWindow.cpp
 msgid ""
 "EXPERT OPTION\n"
 "Set the maximum thread slice runtime (in virtual cycles)"
 msgstr ""
-"专家选项\n"
-"Set the maximum thread slice runtime (in virtual cycles)"
+"专业选项 ⚠\n"
+"线程量子是线程在中断并切换运行的程序前，线程留给程序使用的最大时钟滴答数（以"
+"虚拟的时钟滴答为单位，一个滴答包括若干个时钟周期）"
```
Comment (Fs00) [inline resources/zh/cemu.po:567]: This translation still does not convince me fully.
I wasn't aware of the difference between "clock cycle" and "clock tick", so I would stick with the first one since the word "cycles" is used in the original string (and remove the explanation of the difference between tick and cycle).
I would not use the term "program" for the explanation since Cemu runs only one game at a time (and therefore does not have a concept of "program"). I would rephrase it as "the maximum amount of time in virtual clock cycles for which a thread is allowed to run before it gets paused to let another thread do its work".

File: resources/zh/cemu.po [hunk 26]
```diff
@@ -452,76 +555,36 @@ msgstr "设置CPU模拟模式"
...
-"screen.\n"
-"Auto will enable it for AMD/Intel but disable it for NVIDIA GPUs as a "
-"workaround for a driver bug.\n"
-"\n"
-"Recommended: Auto"
-msgstr ""
-"预编译着色器可减少着色器加载画面所需时间\n"
-"自动：将为AMD与Intel GPU启用而对NVIDIA GPU禁用以解决驱动错误\n"
-"建议选项：自动"
+msgstr "个时钟滴答"
```
Comment (Fs00) [inline resources/zh/cemu.po:571]: As said above, use "clock cycles" instead of "ticks".

File: resources/zh/cemu.po (hunk 25 above)
Comment (Genglxy) [reply resources/zh/cemu.po:567]: Yes. Because in Cemu, the original text here is "Thread quantum 20000 cycles", in order to make it more fluent in Chinese, I added is to make it conform to Chinese reading habits. If you think this is inappropriate, it can be corrected, but will not read fluent.

File: resources/zh/cemu.po (hunk 24 above)
Comment (Genglxy) [reply resources/zh/cemu.po:558]: Yes. Because in Cemu, the original text here is "Thread quantum 20000 cycles", in order to make it more fluent in Chinese, I added is to make it conform to Chinese reading habits. If you think this is inappropriate it can be corrected, but will not read fluent.

File: resources/zh/cemu.po (hunk 25 above)
Comment (Genglxy) [reply resources/zh/cemu.po:567]: In the explanation you sent me earlier, you said "'cycles' here refers to the'clock ticks' mentioned in the explanation above." Regarding this, the result I found is that they are indeed different, the search results found as follows:

> clock ticks refer to the main system clock. It is the smallest unit of time recognized by the device. clock cycle is the time taken for a full processor pulse to complete. this u can recognize by your cpu cpeed given in Hz. a 2GHz processor performs 2,000,000,000 clock cycles per second.

This is why I translate it as clock ticks.
For the latter, my previous understanding was that the thread needs to suspend Cemu's work to complete other Windows programs, and leave Cemu a certain number of clock ticks, so I used the term program.
Is there a problem with my understanding? I will change it later.

Comment (Genglxy) [reply resources/zh/cemu.po:567]: What do you think of this translation?
线程时间片长度为 xxxx 个周期
专业选项 ⚠
设置线程（以时钟滴答为单位）运行时间片的最大值
I asked some friends, Wikipedia mentioned:

The period of time for which a process is allowed to run in a preemptive multitasking system is generally called the time slice or quantum. The scheduler is run once every time slice to choose the next process to run. The length of each time slice can be critical to balancing system performance vs process responsiveness-if the time slice is too short then the scheduler will consume too much processing time, but if the time slice is too long, processes will take longer to respond to input.

An interrupt is scheduled to allow the operating system kernel to switch between processes when their time slices expire, effectively allowing the processor's time to be shared between a number of tasks, giving the illusion that it is dealing with these tasks in parallel (simultaneously). The operating system which controls such a design is called a multi-tasking system.

So translate quantum into time slice, the more commonly used term.

**Comment by Genglxy** [comment]
> There is something to fix in regard to the new translations of thread quantum-related strings, the rest is good.
> What about the translation of "title"? Did you decide to stick to the generic term?

Regarding the translation of title, the result of my discussion with several friends is that the 档案 is still the most suitable Chinese translation currently. Because in Chinese, the meaning of title here is corresponding to multiple words, we can only choose the most important one.  The translation into "档案" fits the context of the time in every place where the title is used, and any other substitute words are not good enough.  If you find it unacceptable, we can only use the title itself to put it here, and this will cause more misunderstandings, because at the English level of most Chinese people, title only means title.

File: resources/zh/cemu.po (hunk 25 above)
Comment (Fs00) [reply resources/zh/cemu.po:567]: > In the explanation you sent me earlier, you said "'cycles' here refers to the 'clock ticks' mentioned in the explanation above." Regarding this, the result I found is that they are indeed different

Yes, in fact initially I wasn't aware of this difference. But the key thing to understand is that Cemu says "**virtual** clock cycles", meaning the cycles of the emulated CPU. Therefore it's not related to OS ticks or other programs in the system.

> For the latter, my previous understanding was that the thread needs to suspend Cemu's work to complete other Windows programs, and leave Cemu a certain number of clock ticks, so I used the term program.

When I say thread, I refer to a piece of code running in parallel. Cemu needs to emulate the behavior of the Wii U scheduler, which needs to execute several threads concurrently (these threads are spawned by the running game) but only has 3 cores (which are emulated by Cemu) to do it.

In your Wikipedia quote you have this sentence:
> An interrupt is scheduled to allow the **operating system kernel** to switch between **processes** when their time slices expire

In Cemu context, that would be:

> An interrupt is scheduled to allow the **Cemu scheduler** to switch between **threads of the running game** when their time slices expire

I hope this is a bit more clear now.

Comment (Fs00) [reply resources/zh/cemu.po:567]: > 线程时间片长度为 xxxx 个周期
专业选项 ⚠
设置线程（以时钟滴答为单位）运行时间片的最大值

I would still use "quantum" in the first line. Also, you must specify that cycles are **virtual** (if you want, you can rephrase it as "cycles of the emulated CPU").
Use always "cycles" instead of "ticks" because that's the term chosen by Cemu devs (and they know better than us how Cemu works internally).

Comment (Genglxy) [reply resources/zh/cemu.po]: Okay, I think I probably understand. I have uploaded a new translation, which also fixes a few minor issues that were not discovered before. If you have time, please take a look.

**Comment by Fs00** [comment]
Thanks for all your hard work!

**Comment by Genglxy** [comment]
> Thanks for all your hard work and patience!

Anytime. In the current latest translation submission, I have compared the results of all past translators, including myself, and selected the most appropriate translation, so this is the result of the combined efforts of many people. See you in the next version!
